src/Json.Schema.ToDotNet.Cli/Json.Schema.ToDotNet.Cli.csproj

```diff
@@ -3,8 +3,9 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFrameworks>netcoreapp2.0</TargetFrameworks>
-    <AssemblyNameOverride>JsonSchemaToDotNet</AssemblyNameOverride>
+    <AssemblyNameOverride>Microsoft.Json.Schema.ToDotNet.Cli</AssemblyNameOverride>
     <RootNamespaceOverride>Microsoft.Json.Schema.ToDotNet.CommandLine</RootNamespaceOverride>
```

Comment:
>RootNamespaceOverride [](start = 5, length = 21)

Now that the assembly has a name that matches the project name, we don't need these overrides at all. We can use the defaults defined in `build.props`. We'd just need to change the namespaces in the two source files in this directory.

Reply:
@michaelcfanning Could you explain why? It's a quick fix and, in fact, with the fix in place, I can even remove the "override" logic in `build.props` (it's only there because for the tools, the exe name didn't match the project name).

Would you like me to take care of it in a separate PR?

---
In reply to: [213834909](https://github.com/Microsoft/jschema/pull/30#discussion_r213834909) [](ancestors = 213834909)

Reply:
I just didn't understand your comment.

---
In reply to: [213843565](https://github.com/Microsoft/jschema/pull/30#discussion_r213843565) [](ancestors = 213843565,213834909)